### PR TITLE
Upgrade to latest threejs (r164)

### LINKED ISSUE #4786: Upgrade to latest version of threejs (r164.1)
### Description

Upgrade to the latest version of ThreeJS

### Version

- model-viewer: v3.5.0

### Browser Affected

- [x] Chrome, version: xx.x.xxxx.xx
- [x] Edge
- [x] Firefox
- [x] IE
- [x] Safari

### OS

- [x] Android
- [x] iOS
- [x] Linux
- [x] MacOS
- [x] Windows

### AR

- [ ] WebXR
- [ ] SceneViewer
- [ ] QuickLook

### PR DESCRIPTION
**Pull request by aardvarkk**
### Reference Issue

Fixes #4786

### CODE REVIEW

**Review by elalish — APPROVED** [review]
Thanks!

File: packages/model-viewer/src/features/ar.ts [hunk 1]
```diff
@@ -446,7 +446,7 @@ configuration or device capabilities');
       model.position.copy(target.position);
       model.updateWorldMatrix(false, true);
 
-      const arraybuffer = await exporter.parse(model);
+      const arraybuffer = await exporter.parseAsync(model);
```
Comment (elalish) [inline packages/model-viewer/src/features/ar.ts:449]: Actually, I believe there's another `exporter.parse` in `scene-graph.ts` as well - could you update that as well for consistency?

Comment (aardvarkk) [reply packages/model-viewer/src/features/ar.ts:449]: Yes, I'll take a look.

Comment (aardvarkk) [reply packages/model-viewer/src/features/ar.ts:449]: @elalish I don't believe the call in `scene-graph.ts` needs to be altered as it's using the synchronous callback-based version of `parse`. `parse` previously used to support both a callback and Promise-based overload, but now the asynchronous version is called `parseAsync`. `ar.ts` await-ed the return, so it needed to be adjusted to use the new async version, but the call to `parse` in `scene-graph.ts` is callback-based.

Comment (elalish) [reply packages/model-viewer/src/features/ar.ts:449]: okay, thanks.

**Pull request merged**